src/storage/btree_load.c

```diff
@@ -264,6 +260,40 @@ static int online_index_builder (THREAD_ENTRY * thread_p, BTID_INT * btid_int, H
 				 int ib_thread_count, TP_DOMAIN * key_type);
 static bool btree_is_worker_pool_logging_true ();
 
+typedef struct
+{
+  OID class_oid;		/* Class OID value in this sorted item */
+  OID rec_oid;			/* OID value in this sorted item */
+  MVCC_REC_HEADER mvcc_header;
+
+  DB_VALUE this_key;		/* Key value in this sorted item (specified with in_recdes) */
+  BTREE_MVCC_INFO mvcc_info;
+  bool is_btree_ops_log;
+
+  OID orig_oid;
+  OID orig_class_oid;
+  MVCC_REC_HEADER orig_mvcc_header;
+} S_PARAM_ST;
+
```

Comment:
the below functions is related to "btree load", so it looks bettter of the name to start with "btree_load" or "bt_load"

Reply:
That's a good suggestion.
I'll fix it.